.github/workflows/go.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: [1.16, 1.17]
+        go: [1.17, 1.18]
```

Comment:
You'll also need to update the linter below to the latest version (1.45.2). That's why CI failed on Go 1.18.